### PR TITLE
update export to use new datamodels

### PR DESCRIPTION
**Pull request by sherwoodf**
Ticket: https://app.clickup.com/t/8696augk3

Updating export to use new models. Also updates output, and copes with api pagination.

I haven't gotten to test it fully, as there's only 1 study and 1 dataset with some file references in the api, so may need a further round of updates.

I'm also not sure i've gotten the default_open flag working correctly with the new nesting system - again, this will be easier to re-visit once there's more studies in the api to test with, but this seemed like already quite a big change, and i didn't want such a specific fix to get lost in this PR.

### CODE REVIEW

File: bia-export/bia_export/website_export/images/models.py [hunk 1]
```diff
@@ -1,43 +1,51 @@
...
+    subject: Specimen = Field(
+        description="""The specimen that was prepared for and captured in the field of view of the image.""",
+        default=None,
+    )
+    protocol: List[Protocol] = Field(
+        description="""The processes involved in the experimental aquisition of the image.""",
+        default_factory=list,
+    )
+    annotation_method: List[AnnotationMethod] = Field(
+        description="""The processes involved in the experimental aquisition of the image.""",
```
Comment (AybukeKY) [inline bia-export/bia_export/website_export/images/models.py:30]: The description seems to be a copy-paste.

**Review by AybukeKY — COMMENTED** [review]
I couldn't find any export for the AnnotationData class which is still in the models and holds the protocol, coverage etc info.

File: bia-export/bia_export/website_export/images/models.py [hunk 2]
```diff
@@ -1,43 +1,51 @@
...
+    acquisition_process: List[ImageAcquisitionProtocol] = Field(
+        description="""The processes involved in the experimental aquisition of the image.""",
+        default_factory=list,
+    )
+    subject: Specimen = Field(
+        description="""The specimen that was prepared for and captured in the field of view of the image.""",
+        default=None,
+    )
+    protocol: List[Protocol] = Field(
+        description="""The processes involved in the experimental aquisition of the image.""",
```
[the remark this answers is not in the document]
Comment (sherwoodf) [reply bia-export/bia_export/website_export/images/models.py:26]: Ah yep, will fix this.

**Comment by sherwoodf** [comment]
> I couldn't find any export for the AnnotationData class which is still in the models and holds the protocol, coverage etc info.

Ah, i don't know where we want for this for the website, so i'm not sure how we want to export it. Do we want a table for these similar to files?

**Comment by sherwoodf** [comment]
@AybukeKY Fixed the field descriptions. Since we don't export File References for tables either, i don't think it's worth me writing export for non-image annotation data right now - especially since you're working on having tables get similar information from the API.

**Review by AybukeKY — APPROVED** [review]

**Pull request merged**